static/js/message_list_view.js

```diff
@@ -1106,8 +1106,13 @@ MessageListView.prototype = {
 
     _maybe_format_me_message: function (message_container) {
         if (message_container.msg.is_me_message) {
-            // Slice the '<p>/me ' off the front, and '</p>' off the end
-            message_container.status_message = message_container.msg.content.slice(4 + 3, -4);
+            // Slice the '<p>/me ' off the front, and '</p>' off the first line
+            // 'p' tag is sliced off to get sender in the same line as the
+            // first line of the message
+            var msg_content = message_container.msg.content;
+            var p_index = msg_content.indexOf('</p>');
+            message_container.status_message = msg_content.slice(7, p_index) +
+                                                msg_content.slice(p_index + 4);
```

Comment:
Can you explain why this won't result in unmatched tag issues?  I guess the idea is we're removing the `<p>` at the start and the first `</p>` (which must be corresponding) wherever it appears?